website_animate/static/src/js/o_animate.frontend.js

```diff
@@ -87,18 +91,31 @@ var WebsiteAnimate = {
 
     // Start animation and/or update element's state
     start_animation: function ($el) {
+        var self = this;
         // force the browser to redraw using setTimeout
         setTimeout(function () {
+            self.toggleOverlayX(true);
             $el
             .css({"animation-play-state": "running"})
             .addClass("o_animating")
             .one('webkitAnimationEnd oanimationend msAnimationEnd animationend', function (e) {
                 $el.addClass("o_animated").removeClass("o_animating");
+                self.toggleOverlayX(false);
                 $(window).trigger("resize");
             });
         });
     },
 
+    // hide the horizontal scrollbar (on the #wrapwrap)
+    toggleOverlayX: function (hide) {
```

Comment:
The common convention for "toggle" function is that the argument is boolean which is true to "show"